src/jwt.rs

```diff
@@ -44,9 +47,16 @@ mod tests {
     }
 
     #[test]
-    fn invalid_jwt() {
+    fn empty_jwt() {
         let e = decode_jwt("").unwrap_err();
-        let _err_msg = "Failed to parse Auth Token".to_owned();
+        let _err_msg = "Malformed Auth Token".to_owned();
+        assert!(matches!(e, MomentoError::ClientSdkError(_err_msg)));
+    }
+
+    #[test]
+    fn invalid_jwt() {
+        let e = decode_jwt("wfheofhriugheifweif").unwrap_err();
+        let _err_msg = "Please make sure entered auth token matches your auth token".to_owned();
```

Comment:
Maybe reword `Please double check that your auth token was entered correctly` would get @danielamiao buy in here as well.

Reply:
`Could not parse token. Please ensure a valid token was entered correctly.`

I just thought it might be helpful to state the problem upfront.